src/validation.cpp

```diff
@@ -5614,11 +5613,8 @@ bool Chainstate::ResizeCoinsCaches(size_t coinstip_size, size_t coinsdb_size)
     return ret;
 }
 
-//! Guess how far we are in the verification process at the given block index
```

Comment:
Why delete this comment?

Reply:
Yes, I suggested these in https://github.com/bitcoin/bitcoin/pull/31177#discussion_r1940010147.